tests/test_tifs.py

```diff
@@ -14,14 +14,35 @@
 _API = {'main': API_URL, 'develop': API_TEST_URL}
 
 
-def _get_tif_tolerances(file_name):
-    tif_type = file_name.name.split('_')[-1]
-    if tif_type == 'phase.tif':  # InSAR
-        return 0.0, 0.0
-    if tif_type == 'area.tif':  # RTC
-        return 2e-05, 0.0
-    if tif_type in ['VV.tif', 'VH.tif', 'HH.tif', 'HV.tif']:  # RTC
+def _get_tif_tolerances(file_name: str):
+    """
+    return the absolute and relative tolerances for the tif comparisons.
+    Comparison will use `numpy.isclose` on the back end:
+        https://numpy.org/doc/stable/reference/generated/numpy.isclose.html
+    Comparison function:
+         absolute(a - b) <= (atol + rtol * absolute(b))
+
+    returns: rtol, atol
```

Comment:
ugg, this tripped me up being `rtol, atol` and not `atol, rtol` when looking at the comparison function as written in Numpy's docs. 

However, the kwargs to go `np.isclose( rtol=rtol, atol=atol)` in the docs and that's the way out `compare.values_are_close()` function is written (following the fn signature of `isclose`). 

I don't really like the comparison function being written in a different order than the kwargs/fn returns, but they are out of line in the Numpy docs... Not sure what to do about that:
* re-order the numpy comparison function in our docs?
* re-order the retuns/kwargs since they are passed as kwargs and it shouldn't matter?
* ignore it and grumble about it every time I look at it?